modules/ting_search/ting_search.module

```diff
@@ -947,6 +971,8 @@ function ting_search_submit($form, &$form_state) {
 
   }
 
+  // TODO BBS-SAL: Should this come from the provider? Its also present in the
+  // alter for the form.
```

Comment:
Perhaps this could use the common fields we have defined?

Reply:
👍  - I'll keep it in mind for the facet issue.